src/systems.rs

```diff
@@ -133,7 +135,9 @@ pub fn particle_spawner(
             spawn_point.translation.z = particle_system
                 .z_value_override
                 .as_ref()
-                .map_or(0.0, |jittered_value| jittered_value.get_value(&mut rng));
+                .map_or(transform.translation.z, |jittered_value| {
```

Comment:
Will this be correct for  `ParticleSpace::Global` if the particle system is parented under some other object? Shouldn't it be `origin_pos.z` instead?